File: typescript/6kyu/Stop_gninnipS_My_sdroW!.ts
export const spinWords = (words: string): string => {
  const wordArr = words.split(" ");
  for (let i = 0; i < wordArr.length; i++) {
    if (wordArr[i].length >= 5) {
      const revWord = wordArr[i].split("").reverse().join("");
      wordArr.splice(i, 1, revWord);
    }
  }
  return wordArr.join(" ");
};

File: python/6kyu/Stop_gninnipS_My_sdroW!.py
def spin_words(sentence):
    word_list = sentence.split(" ")
    for word in word_list:
       if(len(word) >= 5):
          rev_word = word[::-1]
          idx = word_list.index(word)
          word_list[idx] = rev_word
    return " ".join(word_list)         